bot.py

```diff
@@ -1341,7 +1341,17 @@ async def auto_link(ctx):
 
     await ctx.respond(content=message_content, ephemeral=True)
 
-
+@bot.translate_command(
+    description = "Automatically translate any language into another using Google Translate API", aliases = ['tr']
+)
```

Comment:
I can't find `@bot.translate_command` in the list of valid command decorators. To see the valid decorators, see [here](https://docs.pycord.dev/en/master/api.html#bot). Did you mean `@bot.slash_command`?

Additionally, this command would be best as a [Message Context Command](https://docs.pycord.dev/en/master/api.html#messagecommand). It would be awesome if you could implement it this way instead. You could even add a new section of the file like `-------- MESSAGE CONTEXT COMMANDS --------` to keep it organized if you want, too.